cli/cdsctl/workflow_as_code.go

```diff
@@ -396,15 +401,15 @@ func workflowInitRun(c cli.Values) error {
 	}
 
 	if err := workflowTarReaderToFiles(dotCDS, tr, true, true); err != nil {
-		return err
+		return fmt.Errorf("error on workflowTarReaderToFiles:%s", err)
```

Comment:
this message is not user-friendly, you should fix workflowTarReaderToFiles to raise a nice error
you should use %v to print an error (the `default format` as explained in https://golang.org/pkg/fmt/)

Reply:
fixed